### PR TITLE
hey hey hey,, namaste

### PR DESCRIPTION
**Pull request by lol-catz**

### CODE REVIEW

**Review by talaviram — CHANGES_REQUESTED** [review]
I saw you merge from the main branch. Instead, it's cleaner to rebase your branch / PR this will keep commits ordered better without extra noise.

File: source/jucePlugin/VirusController.cpp [hunk 1]
```diff
@@ -945,7 +945,7 @@ namespace Virus
 
 	juce::String numToKeytrigger(float v, Parameter::Description)
 	{
-		return juce::roundToInt(v) == 0 ? "Off" : "Keytrigger Phase " + juce::String(juce::roundToInt(v));
+		return juce::roundToInt(v) == 0 ? "Off" : juce::String(juce::roundToInt(v));
```
Comment (talaviram) [inline source/jucePlugin/VirusController.cpp:948]: What's the fix here? Remember that exposed parameters are also used for automation lanes.

File: source/jucePlugin/VirusController.cpp [hunk 2]
```diff
@@ -476,7 +476,7 @@ namespace Virus
 		case 5:  return "S&G";
 		case 6:  return "Wave3";
 		case 7:  return "Wave4";
-		case 9:  return "Wave5";
```
Comment (talaviram) [inline source/jucePlugin/VirusController.cpp]: This commit can be squashed to previous instead. The nice art of have a branch / PR

**Review by talaviram — CHANGES_REQUESTED** [review]

File: source/jucePlugin/VirusController.cpp (hunk 1 above)
Comment (lol-catz) [reply source/jucePlugin/VirusController.cpp:948]: hmm.. I didnt realize that automation issue i just created here..  i dont know the correct answer for this..     the problem with the current  implemtation is the value does not show up in the box next to the slider

File: source/jucePlugin/VirusController.cpp (hunk 2 above)
Comment (lol-catz) [reply source/jucePlugin/VirusController.cpp]: i see.  thank you for this info.  immaa git noob... need to study git docs .. sorry for the mess.

File: source/jucePlugin/VirusController.cpp (hunk 1 above)
Comment (talaviram) [reply source/jucePlugin/VirusController.cpp:948]: Start playing with git rebase -i and the oldest commit you'd like to start "rewriting history" for.

Just remember you'll need to force push to this branch afterward as they'll diverge.

Comment (lol-catz) [reply source/jucePlugin/VirusController.cpp:948]: noted..  will def be spending some time this week learning to properly work git...  i know there are a lot of gotchas if you don't think things through ahead of time.

Comment (lol-catz) [reply source/jucePlugin/VirusController.cpp:948]: back to this specific edit though  what to you propose  the correct solution is?

**Pull request closed without merging**